tests/load-tests/run-max-concurrency.sh

```diff
@@ -73,15 +73,15 @@ else
         index=$(printf "%04d" "$t")
         cp -vf "$output_dir/load-tests.json" "$output_dir/load-tests.max-concurrency.$index.json"
         cp -vf "$output_dir/load-tests.log" "$output_dir/load-tests.max-concurrency.$index.log"
-        workloadKPI=$(jq '.createApplicationsTimeAvg + .createCDQsTimeAvg + .createComponentsTimeAvg + .integrationTestsRunPipelineSucceededTimeAvg + .runPipelineSucceededTimeAvg + .deploymentSucceededTimeAvg' "$output_dir/load-tests.json")
-        if [ "$workloadKPI" -gt "$threshold" ]; then
+        workloadKPI=$(jq '.workloadKPI' "$output_dir/load-tests.json")
+        if awk "BEGIN { exit !($workloadKPI > $threshold)}"; then
```

Comment:
This is necessary for now we are comparing floating point numbers while before we were comparing integers (`-gt` is for integers only).